alex/applications/PublicTransportInfoCS/data/train_names.expanded.txt

```diff
@@ -0,0 +1,178 @@
+albert einstein	albert einstein
```

Comment:
Shouldn't the Czech Rail provide a list somewhere? Anyway, I wonder whether we should add this to the SVN, or just offer it as a download (same as stops, cities etc.).